spec/index.html

```diff
@@ -172,61 +172,59 @@ <h3>Resources and Statements</h3>
       exists, without explicitly naming it.</p>
   </section>
 
-  <section id="section-quoted-triples">
-    <h3>Quoted Triples</h3>
+  <section id="section-triple-terms">
+    <h3>Triple Terms</h3>
 
-    <p class="issue" data-number="81">
-      As the WG has moved away from the notion of quoted triples, this section needs to be changed such that it does not mention quoted triples anymore but, instead, describes whatever new concept the WG settles on (e.g., <a>triple terms</a>).
-    </p>
-
-    <p>A <dfn>quoted triple</dfn> is an <a>RDF term</a> with the components of
-      an <a>RDF triple</a>, which can be used as the <a>subject</a> or <a>object</a>
+    <p>A <a>triple term</a> is an <a>RDF term</a> with the components of
+      an <a>RDF triple</a>, which can be used as the <a>object</a>
       of another triple.</p>
 
-    <p>A <a>quoted triple</a> is not necessarily asserted, allowing
+    <p>A <a>triple term</a> is not necessarily asserted, allowing
       statements to be made about other statements that may not be
       asserted within an <a>RDF graph</a>.
       This allows statements to be made about relationships that may be contradictory.
-      For a <a>quoted triple</a> to be asserted,
+      For a <a>triple term</a> to be asserted,
       it must also appear in a graph as an <a>asserted triple</a>.</p>
 
-    <p>The following diagram represents a statement with an unasserted <a>quoted triple</a>
-      as the subject.</p>
+    <p>A <a>triple term</a> can be used as the object of a <a>triple</a> with the predicate <code>rdf:reifies</code>.
+      The <a>subject</a> of that <a>triple</a> is called a <dfn>reifier</dfn>.
+      Assertions on the <a>triple term</a> are made using the <a>reifier</a>.</p>
 
-    <figure id="fig-quoted-triple">
-      <a href="quoted-triple.svg">
-        <!-- Source for this file is at https://docs.google.com/drawings/d/1I_QxbUgnQXumXzb8c0WNJHIQ-mtRs2S80dDG6i9aOD8 -->
-        <img src="quoted-triple.svg"
-             alt="An RDF graph containing a triple that references an unasserted quoted triple (with grey background) as the subject"
+    <p>The following diagram represents a statement an a reification of an an unasserted <a>triple term</a>.</p>
```

Comment:
```suggestion
    <p>The following diagram represents a statement and a reification of an unasserted <a>triple term</a>.</p>
```
Not sure about the formulation though, since it mixes statement (interpretation?) and triple (graph?). The triple term is used in another triple to encode a reification?

(The specs may need more clarity in general about whether a triple "encodes a statement", and/or asserting a triple (which "says that some relationship holds") is a statement (which only "corresponds" to a triple).)

Reply:
There's a reason this is still a draft, and I'm hoping for other contributors to help improve the language for clarity and accuracy.

In this case, it is probably better to stick with "triples" and not "statements".